src/EA.Weee.Web/Areas/Admin/Views/ManageEvidenceNotes/ViewAllEvidenceNotes.cshtml

```diff
@@ -39,7 +39,7 @@
                                     <div class="govuk-grid-column-full">
                                         <div class="govuk-grid-row">
                                             <div class="govuk-grid-column-one-half govuk-!-padding-right-0">
-                                                @Html.EditorFor(m => m.ManageEvidenceNoteViewModel, "ObligationTypeStatusFiltersViewModel")
+                                                @Html.EditorFor(m => m.ManageEvidenceNoteViewModel, "StatusFiltersViewModel")
```

Comment:
@NateHogan As we are removing the Obligation / WasteType can we update anywhere in the ManageEvidenceNotesController where the obligation is passed to pass NULL.